tests/Server/Psr7/PsrStreamStub.php

```diff
@@ -29,7 +30,7 @@ class PsrStreamStub implements StreamInterface
      * @see http://php.net/manual/en/language.oop5.magic.php#object.tostring
      * @return string
```

Comment:
we can drop this TH

Reply:
Sorry, I don't understand. What do you mean?

Reply:
sorry, the type hint in phpdoc is redundant as you have added `: string`

Reply:
Ah, gotcha. 34036a4